### DISCUSSION TITLE
ESPN Missing Graphic

### DISCUSSION THREAD

**Discussion by mermelmadness** [vasqued2/ha-teamtracker#111] [General]
Methinks ESPN might be having issues loading the logo icon for Lourdes, lol
![image](https://github.com/vasqued2/ha-teamtracker/assets/38080873/d53c8665-2dfc-4d46-9bdb-4df7b9068d07)

**Comment by vasqued2** [comment]
Yep.  [There have been a couple instances](https://github.com/vasqued2/ha-teamtracker/wiki/Known-Defects-in-the-ESPN-API) where ESPN does not return or returns an old logo for a team.
